docs/modules/ROOT/pages/configuration.adoc

```diff
@@ -1000,6 +1000,16 @@ Sorbet:
   DocumentationExtension: .md
 ----
 
+== Setting the version tracking metadata for cops
+
+Each cop should gain the following additional metadata:
+
+* `VersionAdded` - the RuboCop version in which it was added
+* `VersionChanged` - the RuboCop version in which it was changed in a user-impacting way (new config, updated defaults, etc)
+* `VersionRemoved` - the RuboCop version in which the cop was removed for whatever reason.
+
+Those will be pretty useful for the documentation (so the manual generation has to be enhanced to include them) and keeping track of changes.
```

Comment:
This line probably doesn't make sense in user-facing documentation.

Reply:
Yeah, on the other hand, this might be a meaningful configuration for users who distribute custom cops as a gem. I considered adding it to development.adoc, but thought it might be more appropriate to include it in configuration.adoc for this reason.

At the beginning of the section, I added: "This configuration is particularly useful when custom cops are distributed as a gem."

Reply:
Yeah, we can keep this. The `VersionAdded` would also be useful for the long-demanded and stuck option to enable cops introduced up to a certain version (https://github.com/rubocop/rubocop/issues/11875).